Author: <csmall@debian.org>
Description: Cleaned up the w man page
Make note of the time formats 
Bug-Debian: http://bugs.debian.org/414906
Index: b/w.1
===================================================================
--- a/w.1	2009-11-24 21:00:39.000000000 +1100
+++ b/w.1	2009-11-24 21:00:42.000000000 +1100
@@ -4,26 +4,26 @@
 .SH NAME
 w \- Show who is logged on and what they are doing.
 .SH SYNOPSIS
-.B w \-
-.RB [ husfVo ]
+.B w
+.RB [ \-husfVo ]
 .RI [ user ]
 .SH DESCRIPTION
-.B "w "
+.B w
 displays information about the users currently on the machine,
 and their processes.
 The header shows, in this order,  the current time,
 how long the system has been running,
 how many users are currently logged on,
 and the system load averages for the past 1, 5, and 15 minutes.
-.sp
+
 The following entries are displayed for each user:
 login name, the tty name, the remote host, login time, idle time, JCPU, PCPU,
 and the command line of their current process.
-.sp
+
 The JCPU time is the time used by all processes attached to the tty.  It
 does not include past background jobs, but does include currently
 running background jobs.
-.sp
+
 The PCPU time is the time used by the current process, named in the "what"
 field.
 
@@ -35,7 +35,7 @@
 .TP 0.5i
 .B "\-u "
 Ignores the username while figuring out the current process and cpu
-times.  To demonstrate this, do a "su" and do a "w" and a "w -u".
+times.  To demonstrate this, do a "su" and do a "w" and a "w \-u".
 .TP 0.5i
 .B "\-s "
 Use the short format.
@@ -70,6 +70,19 @@
 process information
 .PP
 
+.SH NOTES
+The output for Idle, JCPU and PCPU times vaires depending on if you use
+the \-o (old style) option or not. These formats can be confusing if you
+switch between the old style and standard.  In the following paragraphs
+days are DD, hours HH, minutes MM, seconds SS and 100ths of seconds CC.
+
+The standard format is DDdays, HH:MMm, MM:SS or SS.CC if the times are
+greater than 2 days, 1hour, or 1 minute respectively.
+
+For the \-o option, the output will be either  DDdays, HH:MM, MM:SSm or
+blank if the times are greater than 2 days, 1 hour or 1 minute
+respectively.
+
 .SH "SEE ALSO"
 .BR free (1),
 .BR ps (1),
